test/pupil-hpa/features/support/redis_helper.rb

```diff
@@ -2,16 +2,17 @@ class RedisHelper
 
   def self.wait_for_prepared_check(school_password,pin)
     p school_password, pin
-    found = false
     begin
       retries ||= 0
       sleep 5
```

Comment:
same as above, use a named variable with the unit in its name

Reply:
Will do as above